google-cloud-vision/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+### 0.32.1 / 2018-09-26
```

Comment:
Bummer I just saw this: The date is incorrect due to copy/paste. (Same with 0.32.0 date).